### ISSUE TITLE
Where to put "stale" explainers

### ISSUE THREAD

**Issue by gregwhitworth** [openui/open-ui#801]
Through the years Open UI has _evolved_ to find its place in how to improve the web platform in collaboration with the other standards bodies. Like other bodies, we do accumulate content that the initial creator was planning on taking further than they ultimately have. This can be for a variety of reasons. A good example of how this can be problematic is @leaverou finding the [breadcrumbs](https://open-ui.org/components/breadcrumb/) _Editor's Draft_ and understandably applying weight to this when it hasn't been touched in over a year.

Additionally, we have pivoted more strongly towards calling out documents "Explainers" which is in line with the definition of "Editor's Draft" through "Community Draft" outlined in our [working mode](https://open-ui.org/working-mode/).

While our [charter](https://open-ui.org/charter/) does state the following:

> - **Research**
>    - Document universal component patterns seen in popular 3rd-party web development frameworks.
>    - Capture commonly-used language for component names and parts, states, behaviors, and transition triggers.
>    - Conduct informal developer-facing user research.
>    - Discover gaps or bugs in what’s possible with current web technology.

and the breadcrumb explainer is possibly a good example of research; the term proposal above it gives it far greater weight. As such. Looking at how other groups handle this, here is what I've found:

CSSWG
----------
The CSSWG has hit a similar issue where _Editor's Drafts_ are implemented against and even discussed in the past making them "Living" for this reason. This lead to some of the W3C process changes to simplify the recommendation process to keep the consensus capabilities in place but enable faster execution from Editor's Draft to recommendation. For documents that are in Editor's Draft and become stale, they "Retire" the document and articulate this clearly to the end user but keep it listed with the [Editor's Draft listings](https://drafts.csswg.org/). A good example of this is [CSS TV](https://drafts.csswg.org/css-tv/), here is the not
<img width="1402" alt="Screenshot 2023-08-16 at 10 14 23 AM" src="https://github.com/openui/open-ui/assets/865244/82ea2434-8081-44b6-947f-21d1cc9de85b">
ice that is shown:

WHATWG
----------
WHATWG is simple in that it is a "living" standard so if the PR lands it is considered "standardized" in the HTML specification and safe for anyone to implement against; thus nothing is ever "stale".

TC39
----------
TC39 stages are fairly straightforward and understood by anyone that keeps track of the specifications making their way through TC39. If something becomes "stale" it can be demoted down a stage but it is not removed from being tracked or available.

TLDR;
-------
What does Open UI want to do in order to alleviate confusing readers that proposals that are dormant are no longer being pursued but also keep the information available for future discovery. To add another wrinkle to this, how do we want to handle V2 versions of explainers. We purposefully punted on `hint` from the `popover` API and created a new version. Should this require a new "fork" of the Explainer like the CSSWG does or should we keep the explainer living? When an explainer has landed as a standard, do we want to demark this in some way via a label in addition to our resolution of linking to the spec in the explainer?

**Comment by gregwhitworth** [comment]
This is now completed by having two sub-navs that are active vs not-active

<img width="326" alt="image" src="https://github.com/openui/open-ui/assets/865244/5915d741-fc95-476d-9c4a-7ba06084c6bc">
